packages/trader/src/Modules/CFD/Containers/props.types.ts

```diff
@@ -37,3 +39,25 @@ export type TPasswordResetAndTradingPasswordManager = {
     platform: 'mt5' | 'dxtrade';
     account_group: 'real' | 'demo';
 };
+
+export type TResetPasswordIntent = {
+    current_list: Array<DetailsOfEachMT5Loginid> & Record<string, DetailsOfEachMT5Loginid>;
+    children({ ...props }): React.ReactElement;
```

Comment:
```suggestion
    children: React.ReactNode;
```
we usually add `React.ReactNode` as the children type. In this way, it can be string, null, ReactElement, etc.

Reply:
If we change 'children({ ...props })'  to 'children' we get this error: "This expression is not callable" since children is returned in one of our component with some params.
Regarding the replacement of ReactElement with ReactNode, the change makes TS to yell at us with the following error ''ResetPasswordIntent' cannot be used as a JSX component. Its return type 'ReactNode' is not a valid JSX element.'
ResetPasswordIntent is not accepting string as one of its types while ReactNode accepts string as a type and we face some issues because of it.